contentpacks/khanacademy.py

```diff
@@ -881,8 +882,6 @@ def apply_dubbed_video_map(content_data: list, dubmap: dict, subtitles: list, la
             if dubmap.get(item.get("youtube_id", "")): # has a dubbed video
                 item.update(dubmap.get(item["youtube_id"]))
                 dubbed_count += 1
-            elif item["kind"] == NodeType.video and item["youtube_id"] not in subtitles: # no subtitles
```

Comment:
> as dicussed with @radinamatic: there are a lot of language packs with 0 video nodes, because they have no dubbed videos and subtitles. The solution we got to, is to mark these language packs as beta, that way it's clear that they don't have widespread support like es and pt.